src/main/java/com/karl/http/HTTPHandler.java

```diff
@@ -10,12 +10,30 @@ public HTTPHandler(Connectable connection) {
     this.connection = connection;
   }
 
+  public String createRequestString(HTTPStatus status, String headers, String body) {
+    return new StringBuilder()
+        .append(Globals.HTTP_VERSION + " ")
+        .append(status)
+        .append(Globals.CRLF)
+        .append(headers)
+        .append(Globals.CRLF)
+        .append(body)
+        .toString();
+  }
+
   public void run() {
     try {
       HTTPRequest request = new HTTPRequestBuilder().withConnection(connection).build();
 
       if (request.methodIs(HTTPMethod.GET) && request.path().equals("/simple_get")) {
-        connection.write("HTTP/1.1 200 OK" + Globals.CRLF + Globals.CRLF);
+        connection.write(createRequestString(HTTPStatus.Ok, "", ""));
+      } else if (request.methodIs(HTTPMethod.GET) && request.path().equals("/simple_get_with_body")) {
+        connection.write(createRequestString(HTTPStatus.Ok, "", "Hello world"));
+      } else if (request.methodIs(HTTPMethod.GET) && request.path().equals("/head_request")) {
+        connection
+            .write(createRequestString(HTTPStatus.MethodNotAllowed, "Allow: HEAD, OPTIONS" + Globals.CRLF, ""));
+      } else {
+        connection.write(createRequestString(HTTPStatus.NotFound, "", ""));
```

Comment:
Saw your comment about needing some sort of routing table which I agree with, but hey this is the simplest solution to get passing tests with only a few cases to support so why not. I much prefer a working solution now that can be refactored later than the _Ideal Solution®_ that might take a while to craft.